resources/migrations/000_migrations.yaml

```diff
@@ -15159,6 +15159,67 @@ databaseChangeLog:
               column:
                 name: action_id
 
+  - changeSet:
+      id: v48.00-007
+      author: qnkhuat
+      comment: Added 0.48.0 - Add revision.most_recent
+      changes:
+        - addColumn:
+            tableName: revision
+            columns:
+              - column:
+                  name: most_recent
+                  type: boolean
+                  defaultValueBoolean: false
+                  remarks: 'Whether a revision is the most recent one'
+                  constraints:
+                    nullable: false
+
+  - changeSet:
+      id: v48.00-008
+      author: qnkhuat
+      comment: Added 0.48.0 - Set revision.most_recent = true for latest revisions
+      changes:
+        - sql:
+            dbms: postgresql,h2
+            sql: >-
+              UPDATE revision r
+              SET most_recent = true
+              WHERE (model, model_id, timestamp) IN (
+                                 SELECT model, model_id, MAX(timestamp)
+                                 FROM revision
+                                 GROUP BY model, model_id);
+          # mysql and mariadb does not allow update on a table that is in the select part
+          # so it we join for them
+        - sql:
+            dbms: mysql,mariadb
+            sql: >-
+              UPDATE revision r
+              JOIN (
+                  SELECT model, model_id, MAX(timestamp) AS max_timestamp
+                  FROM revision
+                  GROUP BY model, model_id
+              ) AS subquery
+              ON r.model = subquery.model AND r.model_id = subquery.model_id AND r.timestamp = subquery.max_timestamp
+              SET r.most_recent = true;
+      rollback: # nothing to do since the most_recent will be dropped anyway
+
+  - changeSet:
```

Comment:
on mysql 5.7. 
<img width="765" alt="Screenshot 2023-08-24 at 17 25 20" src="https://github.com/metabase/metabase/assets/25661381/daadb20b-0566-45a9-b80d-b760a6293e1a">

I was getting [this](https://github.com/metabase/metabase/actions/runs/5951012339/job/16139995753#step:4:1114) error on MySQL and didn't know why. turns out on MySQL 5.7 and mariadb it has a trigger on update...

Reply:
Odd. Can you explain what the ON UPDATE trigger was doing, or where it came from?

Reply:
the ON UPDATE trigger will update `timestamp = now` every time we have an update operation.

In this case we're changing `most_recent = false => true`, and whenever we do that, timestamp gets updated.

> where it came from

I have no idea, we only have this on MySQL 5.7 and mariadb, it might be some default properties when creating timestamp column? idk, I went through the migration and found no place where we define this.

Reply:
Thanks!